client/test/role.go

```diff
@@ -315,17 +315,17 @@ func (r *role) LedgerChannelProposal(rng *rand.Rand, cfg ExecConfig) *client.Led
 		r.log.Panic("Invalid ExecConfig: App does not specify an app.")
 	}
 
-	cfgInitBals := cfg.InitBals()
-	initBals := &channel.Allocation{
-		Assets:   []channel.Asset{cfg.Asset()},
-		Balances: channel.Balances{cfgInitBals[:]},
-	}
-	cfgPeerAddrs := cfg.Peers()
+	peers := cfg.Peers()
```

Comment:
I checked it locally, it works.  
`parts, asset, bals := cfg.Peers(), cfg.Asset(), cfg.InitBals()` 
and then below use  
`parts[:]`.

Reply:
This would work, but it is semantically unclean to provide `parts` when the function asks for `peers`. I wanted to make it clear that there we use the wire addresses, not the wallet addresses.

Reply:
However, what I will do is drop the `parts` completely.